packages/strapi-plugin-graphql/services/type-definitions.js

```diff
@@ -370,60 +371,69 @@ const buildCollectionType = model => {
   }
 
   if (isQueryEnabled(_schema, singularName)) {
-    _.merge(localSchema, {
-      query: {
-        [`${singularName}(id: ID!)`]: model.globalId,
-      },
-      resolvers: {
-        Query: {
-          [singularName]: buildQuery(singularName, {
-            resolver: `${uid}.findOne`,
-            ..._.get(_schema, ['resolver', 'Query', singularName], {}),
-          }),
+    const resolverOpts = {
+      resolver: `${uid}.findOne`,
+      ..._.get(_schema, `resolver.Query.${pluralName}`, {}),
```

Comment:
Is there a reason to switch from a path arrray to a template string ? You have both usage in the file now.

Reply:
When I was workinng on it the line width was too long and it was a way to make it shorter. Also I find it more readable.
I changed all the file to be consistent